source/js/header-navigation.js

```diff
@@ -0,0 +1,25 @@
+
```

Comment:
If we are considering to generate a js file from styleguide we should create modules. So functions like `openNavigation` or variables like `openMenuClasses` don't live in the global scope. There's more info about this [here](https://github.com/getify/You-Dont-Know-JS/blob/master/scope%20%26%20closures/ch5.md)